opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/TracerAutoConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package io.opentracing.contrib.spring.web.autoconfig;
+
+import java.util.logging.Logger;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.opentracing.NoopTracer;
+import io.opentracing.NoopTracerFactory;
+import io.opentracing.Tracer;
+
+/**
+ * @author Pavol Loffay
+ */
+@Configuration
+public class TracerAutoConfiguration {
+    private final static Logger log = Logger.getLogger(TracerAutoConfiguration.class.getName());
+
+    @Bean
+    @ConditionalOnMissingBean(Tracer.class)
+    public Tracer noopTracer() {
+        log.severe("Tracer bean is not configured! Switching to " + NoopTracer.class.getName());
```

Comment:
I still think that this is `error` class log. User wants to use tracing included this auto-configuration but did not read properly documentation and forgot to register tracer.... He restarted app 3x are traces are still not present in zipkin... He would easier spot error log than info.